README.md

```diff
@@ -36,64 +72,123 @@ To successfully receive a payment, these steps should be implemented:
 
 3. The customer returns, and should be satisfied to see that the order was paid and is now being processed.
 
-## Getting started ##
+## Importing the client
```

Comment:
This example doesn’t use Express, and I don’t think it should. That would require us to keep the documentation in sync with Express, and it could give the impression to unexperienced developers that you need Express to use this client.

The example does depend on the Node http module, but it doesn’t include it. I would just use `console.log` and `console.error` to demonstrate the workings of the library. We could just tell the user to redirect to the payment url instead of using `res.writeHead`.

If we want more thorough examples my suggestion would be to create an `examples` folder in the root of this project, and add the relevant examples for Express/Koa/Hapi/Restify/vanilla there. Next.js does this nicely: https://github.com/zeit/next.js/tree/v3-beta/examples